components/html-block/html-block.js

```diff
@@ -286,7 +294,8 @@ class HtmlBlock extends RtlMixin(LitElement) {
 	}
 
 	async _processRenderers(elem) {
-		for (const renderer of getRenderers()) {
+		const renderers = await getRenderers();
```

Comment:
So in the ideal case, the renderers will already be loaded, and the `Promise` returned from `getRenderers` here will already be resolved. But they might not be loaded yet, so we need to wait on that promise. Sound right?

Reply:
That's the idea, yeah.